src/components/hierarchy-bar/hierarchy-bar-collapsed/hierarchy-bar-collapsed.component.ts

```diff
@@ -75,6 +71,14 @@ export class HierarchyBarCollapsedComponent implements AfterViewInit, OnDestroy
         this._resizeService.removeResizeListener(this._elementRef.nativeElement);
     }
 
+    private _update(nodes: HierarchyBarNode[]): void {
```

Comment:
To keep consistency our private functions don't usually have an underscore (unless its public because the view needs access but it should really be private)

Reply:
Yes you are right, thanks. Fixed now.